docs/cedarling/README.md

```
@@ -188,19 +188,19 @@ excessive trust in internal components. The Cedarling aligns with Zero Trust by:
   enforcing the same security rules everywhere.
 
 ### Cedarling and Threat Detection
-Beyond enforcing policies, the Cedarling plays a role in intrusion detection by logging 
-every decision. These logs can be analyzed in a SIEM (Security Information and Event 
+Beyond enforcing policies, the Cedarling plays a role in intrusion detection by logging
+every decision. These logs can be analyzed in a SIEM (Security Information and Event
 Management) system to detect:
 - Unusual access patterns (e.g., a user requesting sensitive data from an unrecognized 
```

markdownlint-cli2 (0.18.1)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)